src/styled/Components/Header.ts

```diff
@@ -0,0 +1,640 @@
+import { Link } from 'react-router-dom';
```

Comment:
same with this, ideally anything with `display: flex` should extend `FlexContainer`

Reply:
done